Changelog.rst

```diff
@@ -609,6 +609,12 @@ pupmod-simp-useradd
 * Set the min and max ``UID`` and ``GID`` based on what is in ``login.defs``
   and default to something sensible for the platform.
 
+
+simp-core
```

Comment:
Have we ever noted the changes to simp-core in the changelog?  If so, there are a whole bunch of changes for this release related to EFI that need to be added.